src/Util.ts

```diff
@@ -184,25 +174,14 @@ export default class W3Util {
     return this.web3.toHex(input);
   }
 
-  private async externalApiNetworkPrice(): Promise<BigNumber> {
+  private async externalApiNetworkPrice(): Promise<GasPriceEstimation> {
     const networkId = await this.getNetworkId();
```

Comment:
I would consider storing network id in some place (maybe config) as it's never changes without config reload. To be  checked though. 

CLI - no network switch
DApp - network switch reloads the page